app/javascript/chat/__tests__/draw.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
+import { axe } from 'jest-axe';
+import Draw from '../draw';
+
+describe('<Draw />', () => {
+  it('should have no a11y violations', async () => {
+    const { container } = render(<Draw />);
+    const results = await axe(container);
+
+    expect(results).toHaveNoViolations();
+  });
+
+  it('should render', () => {
```

Comment:
These tests fail with the following error. The canvas package is required. Run `yarn install canvas -D`.

![image](https://user-images.githubusercontent.com/833231/93110247-1152a900-f683-11ea-80b3-522497277f7a.png)

Reply:
fixed